pisek/utils/util.py

```diff
@@ -54,3 +56,21 @@ def clean_non_relevant_files(accessed_files: set[str]) -> None:
             path = os.path.join(root, file)
             if root in accessed_dirs and path not in accessed_files:
                 os.remove(path)
+
+
+def log_level_mapper(log_level: str) -> Union[int, None]:
```

Comment:
```suggestion
def log_level_mapper(log_level: str) -> int:
```
Sorry, this is a mistake on my side. Can you add typing to `fatal_user_error`?
```py
def fatal_user_error(msg, *args, **kwargs) -> NoReturn:
```

Reply:
updated